SSA/Experimental/Bits/Fast/BitStream.lean

```diff
@@ -429,6 +461,10 @@ instance congr_equiv : Equivalence (EqualUpTo w) where
   symm := equal_up_to_symm
   trans := equal_up_to_trans
 
+theorem BitVec.sub_eq_add_neg : x - y = x + (- y) := by
+  simp only [HAdd.hAdd, HSub.hSub, Neg.neg, Sub.sub, BitVec.sub, Add.add, BitVec.add]
+  simp [BitVec.ofNat, Fin.ofNat', add_comm]
```

Comment:
Should this move into `ForLean.lean` and have there an explicit `{x y : BitVec w}` type annotation.